### ISSUE TITLE
Several countdowns default to 0 seconds

### ISSUE THREAD

**Issue by Pablete1234** [Electroid/PGM#153]
When you use `/cycle`, it does an instant cycle instead of starting the default 30s countdown. This happens on multiple countdowns.
- [ ] `/cycle`
- [ ] `/start`

**Comment by TheMolkaPL** [comment]
This is because provider for `org.joda.time.Duration`s returns `Duration.ZERO` by default.

https://github.com/Electroid/PGM/blob/6aee67bffac1b54a30f14f6a795b6901d19143f2/src/main/java/tc/oc/pgm/commands/provider/DurationProvider.java#L20-L31

**Comment by Pugzy** [comment]
A change to the provider may work as I'm unable to get the below to function with two defaults (one on the map and one on the duration). Not sure if this is the correct way of doing so as it comes with the following exception:

```
app.ashcon.intake.parametric.IllegalParameterException:
Optional<?>, @Default, @Nullable, and @Switch cannot be mixed for parameter #5 of type tc.oc.pgm.map.PGMMap
```


https://github.com/Electroid/PGM/blob/6aee67bffac1b54a30f14f6a795b6901d19143f2/src/main/java/tc/oc/pgm/commands/CycleCommands.java#L26-L32

**Comment by Pablete1234** [comment]
@Pugzy you can't mix default, cause that would mean the parameter is optional. 
I assume, you can't mix an optional with an @Text later, because the command parser wouldn't be able to know what parameter is what

**Comment by KingOfSquares** [comment]
This is solved in #244
